pkgs/development/python-modules/mlxtend/default.nix

```diff
@@ -39,13 +34,33 @@ buildPythonPackage rec {
     joblib
   ];
 
+  checkInputs = [
+    keras
+    pytestCheckHook
+    tensorflow
+  ];
+
+  disabledTests = [
+    # ValueError: fill value must be 0 when converting to COO matrix
+    "test_sparsedataframe_notzero_column"
+  ];
+
+  disabledTestPaths = [
+    # import file mismatch:
+    # imported module 'mlxtend.evaluate.f_test' has this __file__ attribute:
+    #   /build/source/build/lib/mlxtend/evaluate/f_test.py
```

Comment:
Maybe we need to remove the build directory so that we are only using the installed files?